azerothcore/azerothcore-wotlk
# [Zone] Isle of Quel'Danas

Few creatures updated in Deadscar since they were missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Sunwell and then move / fly to Deadscar where you doing daily quests.
